core/src/main/resources/config.yml

```diff
@@ -173,6 +173,11 @@ metrics:
   # UUID of server, don't change!
   uuid: generateduuid
 
+# Microsoft authentication
+msa-authentication:
+  # Specify how many seconds to wait while user authorises Geyser to access their Microsoft account.
```

Comment:
We might need a better description :)

Reply:
Mention something about how it allows users to disconnect while they authenticate.